inc/file/easy_atts_t.cpp

```diff
@@ -24,8 +24,12 @@ TEST_CASE( "Easy attributes")
         INFO("history "<<history);
         std::istringstream ss( history);
         std::tm t = {};
-        ss >> std::get_time( &t, "%Y-%m-%d %H:%M:%S %Z");
+        ss >> std::get_time( &t, "%Y-%m-%d %H:%M:%S");
         CHECK( not ss.fail());
+        std::string timezone;
+        ss >> timezone;
+        INFO( "Timezone "<<timezone);
+        CHECK( timezone.size()); // Unsure how to test this reliably
```

Comment:
Funny, I have a small fix on this in one of my private branches from 1 week ago that looks almost entirely identical. 

You can leave it as it is now, I will need to think a bit more on it but this should not delay this pull-request